app/states/login.html

```diff
@@ -3,12 +3,14 @@
 
         <flash-messages></flash-messages>
         <div ng-include="'templates/server-status.html'"></div>
-        <h1 class="title">Log in</h1>
+        <!-- Removing any sort of ability to log in or recover anything
+        This is done as we update the banner to indicate this service is down while we upgrade 1/12/21 -->
+        <!-- <h1 class="title">Log in</h1> -->
```

Comment:
❓  Just to make sure: our backend server is still offline, right?

Commenting out HTML tags would only be safe if we're sure the backend server is offline, otherwise someone could inspect the page and uncomment it to reactivate the feature.